### PR TITLE
Olwa pr

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

**Review by alexjhawk — DISMISSED** [review]

File: src/com/hms_networks/americas/sc/package.html [hunk 1]
```diff
@@ -11,6 +11,8 @@
 found at https://github.com/hms-networks/sc-flexy-tag-info-lib.
 This library is dependent on the HMS Networks MU Americas Solution Center library for file
 utilities, found at https://github.com/hms-networks/sc-flexy-file-util-lib.
+This library is dependent on the HMS Networks MU Americas Solution Center library for file
+utilities, found at https://github.com/hms-networks/sc-flexy-logger-lib
```
Comment (TomKimsey) [inline src/com/hms_networks/americas/sc/package.html]:
```suggestion
utilities, found at https://github.com/hms-networks/sc-flexy-logger-lib.
```

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: src/com/hms_networks/americas/sc/package.html (hunk 1 above)
Comment (TomKimsey) [reply src/com/hms_networks/americas/sc/package.html]: @oliver-walker-hms  did you see this?

Comment (oliver-walker-hms) [reply src/com/hms_networks/americas/sc/package.html]: Thanks for the heads up

File: src/com/hms_networks/americas/sc/package.html [hunk 2]
```diff
@@ -11,8 +11,10 @@
...
 utilities, found at https://github.com/hms-networks/sc-flexy-file-util-lib.
+This library is dependent on the HMS Networks MU Americas Solution Center library for file
+utilities, found at https://github.com/hms-networks/sc-flexy-logger-lib.
 
 @version 2.0
 @author HMS Networks, MU Americas Solution Center
 </BODY>
-</HTML>
\ No newline at end of file
+</HTML>
```
Comment (TomKimsey) [inline src/com/hms_networks/americas/sc/package.html]: whitespace change

**Review by TomKimsey — CHANGES_REQUESTED** [review]

**Comment by TomKimsey** [comment]
Since @alexjhawk is on vacation and approved already once you fix that ws change and I review again you can merge.

**Review by TomKimsey — APPROVED** [review]

**Pull request merged**